client/src/components/TestSuite/TestSuiteDetails/TestListItem/styles.tsx

```diff
@@ -44,9 +48,12 @@ export default makeStyles((theme: Theme) => ({
     color: 'rgba(0,0,0,0.6)',
   },
   requestUrl: {
-    width: '90%',
-    whiteSpace: 'nowrap',
     overflow: 'hidden',
+    maxHeight: '1.5em',
+    wordBreak: 'break-all',
     textOverflow: 'ellipsis',
```

Comment:
I don't see an ellipsis when the url is too long.

![Screen Shot 2021-12-14 at 10 39 11 AM](https://user-images.githubusercontent.com/15969967/146030100-61d09073-1fed-4dc4-b186-b5aa2fd6c4ad.png)

Reply:
Also, there's still a big scrollbar on the details page:

![Screen Shot 2021-12-14 at 10 44 15 AM](https://user-images.githubusercontent.com/15969967/146030944-e6e7337d-e1c9-4d85-ac67-71969f3e01de.png)

Reply:
Should be fixed now!